usaspending_api/etl/management/commands/load_subawards.py

```diff
@@ -60,17 +62,29 @@ def handle(self, *args, **options):
         else:
             db_cursor = PhonyCursor()
 
-        submission_ids_to_update = []
+        submissions_to_update = []
 
         if options["update_all"]:
             submissions_to_update = SubmissionAttributes.objects.exclude(broker_submission_id__isnull=True)
         else:
-            submission = SubmissionAttributes.objects.filter(broker_submission_id=options['submission_id']).first()
-            if not submission:
-                logger.critical("Submission {} not found in datastore".format(options['submission_id']))
-                return
-            submissions_to_update = [submission]
+            for submission_id in options['submission_id']:
+                sub = SubmissionAttributes.objects.filter(broker_submission_id=submission_id).first()
```

Comment:
Now that the submission loader automatically deletes any submission with the same id, do you think it's time to put a unique index on `broker_submission_id`? I guess that's out of scope, but all of these `first()` queries make me slightly uneasy.

Sometimes when I'm using that pattern, I add a `sort_by(-update_date)` to the query so at least know we're always operating on the same record in the event that something went awry. Do you think that's warranted here?